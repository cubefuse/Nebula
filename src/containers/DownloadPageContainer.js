//
import React from "react";
import { Layout } from "antd";

const { Content } = Layout;

/**
 * Download page for single file / folder downloads.
 */
export default class DownloadPageContainer extends React.Component {
  render() {
    return (
      <Content style={{ padding: "0 24px", minHeight: 280 }}>
        Download Page Container
      </Content>
    );
  }
}
