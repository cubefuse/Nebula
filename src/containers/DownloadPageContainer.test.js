import React from "react";
import "jest-dom/extend-expect";
import { render } from "react-testing-library";
import DownloadPageContainer from "./DownloadPageContainer";

test("can render", () => {
  const node = render(<DownloadPageContainer />);
  expect(node).toMatchSnapshot();
});
