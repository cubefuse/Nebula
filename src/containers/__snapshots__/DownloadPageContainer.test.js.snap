// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render 1`] = `
Object {
  "container": <div>
    <div
      class="ant-layout-content"
      style="padding: 0px 24px; min-height: 280px;"
    >
      Download Page Container
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
